test/functional/repository/find-options/repository-find-options.ts

```diff
@@ -1,5 +1,6 @@
-import "reflect-metadata"
```

Comment:
Why did you remove reflect metadata here? Is it really not necessary?

Reply:
Not necessary. `reflect-metadata` only needs importing once.

Reply:
If `test/utils/test-setup.ts` is loaded correctly by mocha before the tests, this should definitely not be needed. I noticed some files also load `test-setup.ts` by themselves, this should also not be required.

It looks like a bigger cleanup is required for the test setup, so I reverted this for now.